docs/changelog/117933.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 117933
+summary: Change `deprecation.elasticsearch` keyword to `elasticsearch.deprecation`
+area: Infra/Logging
+type: bug
```

Comment:
We should note this as a breaking change, since some users may be relying on the ECS dataset.

Reply:
Thanks @rjernst - I've updated it now, WDYT?